pkgs/tools/filesystems/lizardfs/default.nix

```diff
@@ -16,28 +18,44 @@
 , zlib # optional
 }:
 
-stdenv.mkDerivation rec {
+let
+  # See https://github.com/lizardfs/lizardfs/blob/3.12/cmake/Libraries.cmake
+  # We have to download it ourselves, as the build script normally does a download
+  # on-build, which is not good
+  spdlog = fetchzip {
+    name = "spdlog-0.14.0";
+    url = "https://github.com/gabime/spdlog/archive/v0.14.0.zip";
+    sha256 = "13730429gwlabi432ilpnja3sfvy0nn2719vnhhmii34xcdyc57q";
+  };
+in stdenv.mkDerivation rec {
   name = "lizardfs-${version}";
-  version = "3.11.3";
+  version = "3.12.0";
 
   src = fetchFromGitHub {
     owner = "lizardfs";
     repo = "lizardfs";
     rev = "v${version}";
-    sha256 = "1njgj242vgpdqb1di321jfqk4al5lk72x2iyp0nldy7h6r98l2ww";
+    sha256 = "0zk73wmx82ari3m2mv0zx04x1ggsdmwcwn7k6bkl5c0jnxffc4ax";
   };
 
-  buildInputs = 
-    [ cmake fuse asciidoc libxml2 libxslt docbook_xml_dtd_412 docbook_xsl
+  buildInputs =
+    [ cmake db fuse asciidoc libxml2 libxslt docbook_xml_dtd_412 docbook_xsl
       zlib boost pkgconfig judy pam makeWrapper
```

Comment:
`cmake`, `pkgconfig`, and `makeWrapper` should go in `nativeBuildInputs`.

Reply:
Fixed!